### ISSUE TITLE
filter-option display:inline, width:auto

### ISSUE THREAD

**Issue by bazineta** [snapappointments/bootstrap-select#2261]
Noticed the following in the CSS:
```css
.filter-option {
  position: static;
  display: inline;
  padding: 0;
  width: auto;
}
```
Admittedly, CSS is not just my favorite thing ever, emergency dentistry being considerably higher on the list.

However, seems as if something can be display:inline, or have a width, but not both.

**Comment by caseyjhol** [comment]
Good catch. The `width: auto;` is unnecessary here. I'll need to test it in IE to make sure there aren't any unexpected issues, but it can probably be safely removed.

**Comment by caseyjhol** [comment]
Released in [v1.13.10](https://github.com/snapappointments/bootstrap-select/releases/tag/v1.13.10)!

**Comment by capegreg** [comment]
The change from position: absolute in version 1.13.0 to position: static in version 1.13.9 is breaking the dropdown style. See screenshot. 

![style](https://user-images.githubusercontent.com/1051118/58488038-ac460d80-8136-11e9-9696-4b9f4ccf4a46.png)
